react/react_testing/mocking_callbacks_and_components.md

```diff
@@ -127,6 +127,14 @@ jest.mock('../submission', () => ({ submission, isDashboardView }) => (
 
 We only render the bare minimum to realize the validity of the component we're testing. Next, we set up our props with fake data and mocked functions.
 
+<div class="lesson-note" markdown="1">
+
+#### Note
```

Comment:
Instead of a generic title, we could be more specific. Something along the lines of "Vitest mocks"